modules/mono/glue/cs_files/GodotTaskScheduler.cs

```diff
@@ -6,89 +6,89 @@
 
 namespace Godot
 {
-	public class GodotTaskScheduler : TaskScheduler
-	{
-		private GodotSynchronizationContext Context { get; set; }
-		private readonly LinkedList<Task> _tasks = new LinkedList<Task>();
+    public class GodotTaskScheduler : TaskScheduler
+    {
+        private GodotSynchronizationContext _context{ get; set; }
```

Comment:
Did we agree about this naming convention for private properties?

Reply:
I tried to follow the [style guide](https://godot.readthedocs.io/en/latest/getting_started/scripting/c_sharp/c_sharp_style_guide.html) but it seems that I mistook this property for a field, since it does say that private properties should be PascalCase. I'll change this back.